### ISSUE TITLE
Unable to add individual subscriptions once a catch-all subscription is in place

### ISSUE THREAD

**Issue by gfitzp** [mackworth/cTiVo#353]
I love the ability to record all shows from a particular station (#326 - I use it to save the movies I've recorded off of TCM as well), but I found a functionality limitation that I didn't expect.

It seems that once I have a rule to record all shows in place, even though it's restricted to a single channel, clicking the "Subscribe" button for any other shows doesn't add any subscriptions. If I try to manually add the subscription by entering the series title I get an error: "Invalid Subscription - The subscription pattern may be badly formed, or it may already covered by another subscription."

I'm adding subscriptions to shows that aren't on TCM, so I guess cTiVo is only checking the subscription search string and not taking into account the channel restriction?

**Comment by mackworth** [comment]
You are correct. If the name of a show you're subscribing to matches an existing subscription (with channel or other restrictions notwithstanding), then it won't add the subscription. It probably makes sense to make that match more restrictive, but the easiest fix for you is to open Terminal and cut/paste this string `defaults write com.ctivo.ctivo AllowDups YES` and hit return; . This will let you add duplicate subscriptions and even duplicate downloads to the same show.

**Comment by gfitzp** [comment]
Great, thanks so much! I wasn't sure if it was a bug so had to dig a little further since it didn't seem like it should be preventing me from adding new subscriptions. This fix should be fine for me in the meantime. Thanks again!

**Comment by mackworth** [comment]
Oops.  Forgot the -bool :    `defaults write com.ctivo.ctivo AllowDups -bool YES`
